src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/ProfileController.php

```diff
@@ -210,7 +210,7 @@ public function editAction($profileId, Request $request)
         return $this->render('@PrestaShop/Admin/Configure/AdvancedParameters/Profiles/edit.html.twig', [
             'profileForm' => $form->createView(),
             'layoutTitle' => $this->trans(
-                'Edit: %value%',
+                'Editing profile %value%',
                 'Admin.Catalog.Feature',
```

Comment:
```suggestion
                'Admin.Advparameters.Feature',
```
That's not your fault, the error was already there. But let's take the opportunity to correct it!